runsc/boot/filter/config.go

```diff
@@ -504,6 +504,16 @@ func hostInetFilters() seccomp.SyscallRules {
 				seccomp.EqualTo(unix.SOL_TCP),
 				seccomp.EqualTo(linux.TCP_INQ),
 			},
+			{
+				seccomp.MatchAny{},
+				seccomp.EqualTo(unix.SOL_TCP),
+				seccomp.EqualTo(linux.TCP_MAXSEG),
+			},
+			{
+				seccomp.MatchAny{},
+				seccomp.EqualTo(unix.SOL_TCP),
+				seccomp.EqualTo(linux.TCP_CONGESTION),
+			},
 		},
 		unix.SYS_IOCTL: []seccomp.Rule{
 			{
```

Comment:
You also need to update the seccomp filters for SYS_SETSOCKOPT. Because your change allows host syscalls to SYS_SETSOCKOPT for these socket options.

Reply:
Done